static/src/javascripts/projects/common/modules/commercial/acquisitions-ophan.js

```diff
@@ -0,0 +1,37 @@
+// @flow
+import ophan from 'ophan/ng';
+
+const submitComponentEvent = (componentEvent: OphanComponentEvent) => {
+    ophan.record({ componentEvent });
+};
+
+// Hmmm, what about where the campaign code is different for each button?
```

Comment:
Yeah, I thought we had cases where they were different but I don't think we do any more (since we moved to the `gdnwb_copts_memco` prefix). I think it's actually a good thing to enforce this in the future. Since we always know the product, varying the campaign code between products is redundant